.github/workflows/ci-build.yml

```diff
@@ -81,17 +73,27 @@ jobs:
           path: doc/build/latex/*.pdf
           retention-days: 7
 
-  Release:
+      - name: Deploy to gh-pages
+        if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
```

Comment:
When I introduced these lines, I accidentally imposed our documentation to render only when tagging. Let us remove these @Revathyvenugopal162 so our guidelines are always up to date.

Reply:
Good catch @jorgepiloto!

Reply:
Maybe I am missing something. But in PyMAPDL we just push the docs when release. We use the nightly to update the dev version (based on main branch).

So from my perspective, this is fine. Because you keep the official docs synced with the released version. And for a more up to date docs, you have the "dev" domain.

I mean sure you can do it this way. But I feel like the official docs should match the released version. Although in the dev guide there might be not many differences between main and release.

Reply:
@jorgepiloto I agree with @germa89 . I think it was fine to deploy the "official" doc only for new release.

Reply:
This comes from and old discussion in https://github.com/pyansys/dev-guide/pull/97 just for having as soon as possible our dev guide up to date.

Nevertheless, I understand the "release and tag" concept, as it applies to the rest of PyAnsys projects.

Reply:
The last release is pretty old in fact... dating 25 March.

I don't see the purpose of "releasing" doc repositories. I think they should just be kept up to date as soon as possible, but as @jorgepiloto mentioned, I do agree that it may be the way forward for actual Python libraries.

So, as you see fit. But in case you want to stick to the "release and tag" procedure, you should start considering making a new release soon.

Reply:
I agree about "relasing" doc repos. I think we can release a 0.3.0 now that #64 got merged and documentation shows a new layout.